### PR TITLE
Bugfix/483

### PR DESCRIPTION
**Pull request by jul4ajka**
News component on admin with language panel.

### CODE REVIEW

File: src/components/filter-search-sort/navbar.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+const Navbar = ({ options }) => {
+  const styles = useStyles();
+  const { sortOptions, filterOptions, searchOptions, clearOptions } = options;
+  return (
+    <Grid
+      className={styles.container}
+      container
+      direction='row'
+      justify='flex-start'
+      alignItems='center'
```
Comment (kapoliub) [inline src/components/filter-search-sort/navbar.js:19]: maybe it makes sense to move this styles to useStyles file

File: src/hooks/filters/use-category-filters.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+  };
+  return {
+    sortOptions: {
+      labels: filterLabels.categories.sortLabels,
+      setSorting
+    },
+    filterOptions: {
+      filters,
+      allCategories,
+      buttonTitle: 'Категорія',
```
Comment (kapoliub) [inline src/hooks/filters/use-category-filters.js:55]: move this title to config file

**Review by kapoliub — DISMISSED** [review]

**Review by AndriyLymych — APPROVED** [review]

File: src/components/filter-search-sort/nav-filter-component/nav-filter-component.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+      <Badge
+        badgeContent={filterValues.length}
+        color='error'
+        anchorOrigin={badgePosition}
+      >
+        <FormControl className={styles.formControl}>
+          <InputLabel id='multiple-checkbox-label'>{buttonName}</InputLabel>
+          <Select
+            labelId='multiple-checkbox-label'
+            data-cy={`user-filters-${buttonName}`}
```
Comment (moran711) [inline src/components/filter-search-sort/nav-filter-component/nav-filter-component.js:32]: Please remove `user` from here

File: src/components/filter-search-sort/nav-filter-component/nav-filter-component.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  const styles = useStyles();
+  return (
+    <div className={styles.container}>
+      <Badge
+        badgeContent={filterValues.length}
+        color='error'
+        anchorOrigin={badgePosition}
+      >
+        <FormControl className={styles.formControl}>
+          <InputLabel id='multiple-checkbox-label'>{buttonName}</InputLabel>
```
Comment (moran711) [inline src/components/filter-search-sort/nav-filter-component/nav-filter-component.js:29]: You will have more than one element with this id on your page

File: src/components/filter-search-sort/nav-filter-item/nav-filter-item.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+  filterLabels,
+  filterValues,
+  filterList,
+  filterHandler
+}) => {
+  const formGroupOptions = useMemo(
+    () =>
+      _.map(filterList, (item, idx) => (
+        <MenuItem
+          data-cy={`user-filters-list-${item}`}
```
Comment (moran711) [inline src/components/filter-search-sort/nav-filter-item/nav-filter-item.js:23]: same

File: src/components/filter-search-sort/nav-filter-item/nav-filter-item.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+const NavFilterItem = ({
+  buttonName,
+  filterLabels,
+  filterValues,
+  filterList,
+  filterHandler
+}) => {
+  const formGroupOptions = useMemo(
+    () =>
+      _.map(filterList, (item, idx) => (
```
Comment (moran711) [inline src/components/filter-search-sort/nav-filter-item/nav-filter-item.js:21]: You can import this like `import {map} from 'lodash';`

File: src/components/filter-search-sort/nav-filter-item/nav-filter-item.js [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+        <MenuItem
+          data-cy={`user-filters-list-${item}`}
+          key={item._id}
+          value={item._id}
+        >
+          <Checkbox
+            checked={
+              _.findIndex(filterValues, (filter) => filter === item._id) !== -1
```
Comment (moran711) [inline src/components/filter-search-sort/nav-filter-item/nav-filter-item.js:29]: same

File: src/components/filter-search-sort/nav-sort/nav-sort.js [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+import { useStyles } from './nav-sort.styles';
+
+const NavSort = ({ sortOptions }) => {
+  const [sortValue, setSortValue] = useState(sortOptions.labels[1].value);
+  const styles = useStyles();
+  const { setSorting } = sortOptions;
+
+  const selectOptions = _.map(sortOptions.labels, ({ label, value }) => (
```
Comment (moran711) [inline src/components/filter-search-sort/nav-sort/nav-sort.js:12]: same

File: src/components/forms/news-form/news-form.js [hunk 9]
```diff
@@ -76,8 +58,8 @@ const NewsForm = ({ id, newsArticle, editMode }) => {
 
   const selectFormSchema = () => {
     const formObj = preferredLanguages.reduce((reducer, lang) => {
-      reducer[`${lang}AuthorName`] = Yup.string()
-        .min(6, NAME_MIN_LENGTH_MESSAGE)
+      reducer[`${lang}Authorname`] = Yup.string()
```
Comment (moran711) [inline src/components/forms/news-form/news-form.js:61]: You can use `${lang}AuthorName`

File: src/components/forms/news-form/news-form.js [hunk 10]
```diff
@@ -94,24 +76,17 @@ const NewsForm = ({ id, newsArticle, editMode }) => {
...
-    errors
-  } = useFormik({
+  const { values, handleSubmit, handleChange, touched, errors } = useFormik({
     validationSchema: formSchema,
     initialValues: {
       authorPhoto: newsArticle.author.image || '',
       newsImage: newsArticle.image || '',
-      uaAuthorName: newsArticle.author.name[0].value || '',
-      enAuthorName: newsArticle.author.name[1].value || '',
+      uaAuthorname: newsArticle.author.name[0].value || '',
```
Comment (moran711) [inline src/components/forms/news-form/news-form.js:84]: same

File: src/components/forms/news-form/news-form.js [hunk 11]
```diff
@@ -158,6 +133,20 @@ const NewsForm = ({ id, newsArticle, editMode }) => {
     dispatch(push(config.routes.pathToNews));
   };
 
+  const inputs = [
+    { label: authorName, name: 'authorname' },
```
Comment (moran711) [inline src/components/forms/news-form/news-form.js:137]: You can write here `authorname`

File: src/components/forms/news-form/news-form.js [hunk 12]
```diff
@@ -158,6 +133,20 @@ const NewsForm = ({ id, newsArticle, editMode }) => {
...
+    { label: title, name: 'title' },
+    { label: text, name: 'text' }
```
Comment (moran711) [inline src/components/forms/news-form/news-form.js:139]: text need to have `isEditor: true` field here

File: src/pages/users/components/admin-tab/admin-tab.js [hunk 13]
```diff
@@ -53,6 +54,7 @@ const AdminTab = (props) => {
 
   return (
     <>
+      <UserNavbar />
```
Comment (moran711) [inline src/pages/users/components/admin-tab/admin-tab.js:57]: you can use your custom `FilterNavbar` here

**Review by moran711 — APPROVED** [review]

**Review by AndriyLymych — APPROVED** [review]

**Pull request merged**